src/styles/blocks/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
```

Comment:
`flex` for the card is redundant - all block elements will be displayed on the new line by default